docs/api/app.md

```diff
@@ -403,6 +408,36 @@ Returns:
 Emitted when the renderer process unexpectedly dissapears.  This is normally
 because it was crashed or killed.
 
+#### Event: 'child-process-gone'
```

Comment:
This is awkward naming because of the child_process module in node.  Is there another name we can use for this that wouldn't conflict with that existing concept

Reply:
IMO this is a different enough context that the namespace collision doesn't bother me, and it's valuable i think to match Chromium's naming where possible.